Libraries/LibWeb/Editing/CommandNames.cpp

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2024, Jelle Raaijmakers <jelle@ladybird.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibWeb/Editing/CommandNames.h>
+
+namespace Web::Editing::CommandNames {
+
+#define __ENUMERATE_COMMAND_NAME(name) FlyString name;
+ENUMERATE_COMMAND_NAMES
+#undef __ENUMERATE_COMMAND_NAME
+FlyString delete_;
+
+void initialize_strings()
+{
+    static bool s_initialized = false;
+    VERIFY(!s_initialized);
+
+#define __ENUMERATE_COMMAND_NAME(name) name = #name##_fly_string;
+    ENUMERATE_COMMAND_NAMES
+#undef __ENUMERATE_MATHML_TAG
```

Comment:
Shouldn't `__ENUMERATE_COMMAND_NAME` be undefined here? I don't see any use of `__ENUMERATE_MATHML_TAG` anywhere in the file and other uses of this pattern also don't do that

Reply:
Copy/paste error, feel free to PR!

Reply:
Done: https://github.com/LadybirdBrowser/ladybird/pull/2665